lib/src/screens/home_screen.dart

```diff
@@ -1,10 +1,62 @@
 import 'package:flutter/material.dart';
```

Comment:
should we call this file is "NavigationScreen"?

Reply:
I renamed it to NavigationScreen

Reply:
I renamed it to NavigationScreen